l2-frontend/src/utils.ts

```diff
@@ -368,24 +368,7 @@ export const convertSubjectNameToTitle = (object: any, subjectName: string | nul
   console.log(obj);
   // eslint-disable-next-line no-console
   console.log(subjectName);
-
-  if (!obj.SNILS && obj['СНИЛС']) {
-    obj.SNILS = obj['СНИЛС'];
-  }
-
-  let result = `НЕТ СНИЛС ${name}`;
-  if (obj.CN) {
-    if (obj.T && obj.SN && obj.G) {
-      let CN = obj.CN.replace('"""', '""');
-
-      if (CN.length > 1 && CN[0] === '"' && CN[CN.length - 1] === '"') {
-        CN = CN.slice(1, -1);
-      }
-      CN = CN.replace('""', '"');
-      result = `${!obj.SNILS ? 'НЕТ СНИЛС ' : ''}${obj.SN} ${obj.G}${obj.SNILS ? `, ${obj.SNILS}` : ''} — ${obj.T} — ${CN}`;
-    }
-  }
-  return result;
+  return `${obj.SN} ${obj.G} ${obj.SNILS} ${obj.T ? obj.T : ''}`;
```

Comment:
Лучше примерно так (но стоит проверить)
Что бы не было лишних пробелов и исключить случаи с другими пустыми значениями
```suggestion
  return [obj.SN, obj.G, obj.SNILS, obj.T].filter(Boolean).join(' ');
```

Reply:
return [obj.SN, obj.G, obj.SNILS, `${obj.T ? obj.T : ''}`].filter(Boolean).join(' ');

T - не всегда присутствует